API/controllers/user.controller.js

```diff
@@ -3,16 +3,28 @@ import userData from '../utils/data';
 class userController {
   static signIn(req, res) {
     const { email, password } = req.body;
-    const userA = userData.filter(user => user.email === email && user.password === password);
-    if (!userA) {
+    const user = userData.find(u => u.email === email);
```

Comment:
the use of `u` is not very descriptive and makes the code hard to understand. change it to a more descriptive name

Reply:
I will modify that right away.